### PR TITLE
95918 add sub navigation styles to style guide

### PR DESCRIPTION
**Pull request by joebri**

### CODE REVIEW

File: styles/nav.scss [hunk 1]
```diff
@@ -121,7 +121,7 @@ $daptiv-subnav-tab-h-pad: 14px;
 
     .daptiv-subnav-tab {
         color: $daptiv-subnav-font-color;
-        display: inline-block;
+        cursor: pointer;
```
Comment (boskya) [inline styles/nav.scss]: @joebri this should not be needed. We are leaving the cursor to whatever the browser default is.

File: styles/nav.scss [hunk 2]
```diff
@@ -135,6 +135,14 @@ $daptiv-subnav-tab-h-pad: 14px;
             color: $daptiv-subnav-tab-selected;
         }
     }
+    .daptiv-subnav-tab.disabled {
```
Comment (boskya) [inline styles/nav.scss]: @joebri why not
.daptiv-subnav-tab {
   &.disabled {
   }
}

File: styles/nav.scss (hunk 1 above)
Comment (joebri) [inline styles/nav.scss]: @boskya This was addressing issue with missing href attributes in link elements. Have added missing hrefs instead.

File: styles/nav.scss (hunk 2 above)
Comment (joebri) [inline styles/nav.scss]: @boskya No good reason (still coming up to speed on SCSS). Fixed.

**Pull request merged**